src/execution_plan/execution_plan.c

```diff
@@ -108,30 +108,29 @@ Vector* _ExecutionPlan_Locate_References(OpBase *root, OpBase **op, Vector *refe
     /* See if all references been resolved.
      * TODO: create a vector compare function
      * which checks if the content of one is in the other. */
-    int match = Vector_Size(references);
-    size_t ref_count = Vector_Size(references);
+    int match = raxSize(references);
+    size_t ref_count = match;
     size_t seen_count = Vector_Size(seen);
+    
```

Comment:
`seen_count` should definitely be an `int`, may as well make `ref_count` one as well (or change the type of `match`).

Reply:
👍